src/action/client_side_file_finder/mod.rs

```diff
@@ -0,0 +1,8 @@
+// Copyright 2020 Google LLC
+//
+// Use of this source code is governed by an MIT-style license that can be found
+// in the LICENSE file or at https://opensource.org/licenses/MIT.
+
+//! Handler for `client side file finder` action.
+
+mod expand_groups;
```

Comment:
Same. Having a function-specific module seems weird. For now I would put it into something like `util` and then, once it grows enough, split it into more categories.

Reply:
It already grew significantly on my `dev` branch, so I split it into categories. I believe there is a beauty in such a split as this module provides only `fn expand_groups(path: &str) -> Vec<String>` functionality and reader probably will never have to look at this module unless interested in the group expansion.

`Do one thing, and do it well`